pkg/logbridge/logbridge.go

```diff
@@ -0,0 +1,88 @@
+package logbridge
+
+import (
+	"bufio"
+	"io"
+
+	"github.com/square/p2/pkg/logging"
+)
+
+// This type is agnostic of pipes. os.File conforms to io.Reader and io.Writer
+// so it should not be difficult to migrate
+type Bridge struct {
```

Comment:
I don't think it makes sense to package this procedure into its own struct type.  There is only one public method, and you can only call it once.  Instead of this:

    b := logbridge.New(src, dest, 1000)
    b.Run()

what would you think of:

  logbridge.Copy(src, dest, 1000)